voteswap/templates/base.html

```diff
@@ -32,8 +32,10 @@
         </div>
       </div>
     </nav>
-    {% block main %}
-    {% endblock %}
+    <div class="container">
```

Comment:
Can you move the container class to the profile page? It isn't always the case that a single container is suitable for the whole page.